source/getting-started/connection.txt

```diff
@@ -0,0 +1,622 @@
+================
+Connection Guide
+================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+This guide shows you how to connect to MongoDB with the Node.js driver using
+different connection mechanisms as well as how to configure them.
+
+When you connect to a MongoDB server, you provide your credentials and
+specific connection mechanism to the driver. The driver negotiates with the
+server to verify your identity, and the server grants you permissions
+appropriate to your user.
+
+For instructions on how to set up a new user or user permissions on your
+MongoDB server, see the **Authentication** and **Authorization** sections in
```

Comment:
Here, I intentionally wanted to avoid linking out to other pages that were related to DBA-role responsibilities, but also wanted to avoid user confusion as to how permissions are set up. I also did not want to make any assumptions as to what environment they're working in -- whether they are developing locally or are attempting to connect to a managed instance. If it's confusing, I can omit it.

Also, we do not have the `extlinks` module (enables you to define shortcuts like the one you mentioned) in docs-node, so I'm not able to use that feature.

Reply:
The external links are built-in to snooty! https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml#L602

Reply:
Oh, darn. Thanks for that info. I had changed all the `:manual:` instances to external links. ☹️